frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/status-card/utils.ts

```diff
@@ -28,3 +30,16 @@ export const getCephHealthState: PrometheusHealthHandler = (responses = [], erro
   const value = _.get(responses[0], 'data.result[0].value[1]');
   return CephHealthStatus[value] || CephHealthStatus[3];
 };
+
+export const getDataResiliencyState = (results: PrometheusResponse, error: boolean) => {
```

Comment:
you could reuse `PrometheusHealthHandler` type
```suggestion
export const getDataResiliencyState: PrometheusHealthHandler = (results = [], error = []) => {
```